### PR TITLE
LPS-155306 | master

### PR DESCRIPTION
**Pull request by jaydawu**
**Background**
Create automated tests for FDS Filters.

**Test Plan**
**FDSFilters#FilterCanBeEdited**
Given Sample FDS portlet deployed
And entering on a page which uses FDS
When there is a filter already established
And clicking on the Filters Summary
And the filter is edited
Then the results will be displayed updated accordingly to it
And the Filters Summary boxes will be updated accordingly to it

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-155306

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
Taking a look! :detective:

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+
 	macro enableStatusFilters {
-		Click(
-			key_filter = "Status",
-			locator1 = "FrontendDataSet#FILTER_OPTION");
+		if (IsElementPresent(key_filter = "Status", locator1 = "FrontendDataSet#FILTER_OPTION")) {
+			Click(
+				key_filter = "Status",
+				locator1 = "FrontendDataSet#FILTER_OPTION");
+		}
```
Comment (susanchen3) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro:22]: Good addition :+1:

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
-					locator1 = "FrontendDataSet#STATUS_FILTER_CHECKBOX");
-			}
-			else if ("${status}" == "Pending") {
-				Check.checkNotVisibleNoErrors(
-					key_status = "Pending",
-					locator1 = "FrontendDataSet#STATUS_FILTER_CHECKBOX");
-			}
+			Check.checkNotVisibleNoErrors(
+				key_status = "${status}",
+				locator1 = "FrontendDataSet#STATUS_FILTER_CHECKBOX");
```
Comment (susanchen3) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSFilters.macro:27]: Thanks for the edit!

**Review by susanchen3 — APPROVED** [review]
Looks good to me! Thanks Jayda!

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**